tests/unit/python/execution_tree/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2017 Hartmut Kaiser
+#
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+set(tests
+  phylisp_test
+   )
+
+foreach(test ${tests})
+  set(script ${test}.py)
+
+  add_phylanx_python_unit_test("execution_tree" ${test}
+    SCRIPT ${script}
+    FOLDER "Tests/Python/Unit/Execution_Tree"
+    DEPENDS phylanx_py python_setup
+    WORKING_DIRECTORY ${PHYLANX_PYTHON_EXTENSION_LOCATION}
+    ENVIRONMENT "PYTHONPATH=${PHYLANX_PYTHON_EXTENSION_LOCATION}")
+
+  add_phylanx_pseudo_target(tests.unit.python_execution_treee.${test}_py)
+  #add_phylanx_pseudo_dependencies(tests.unit.python_execution_tree tests.unit.python_execution_tree.${test}_py)
```

Comment:
If these pseudo-dependencies are not in place the test will not be executed by 'make tests'

Reply:
But they are executed by "make tests." I checked.

Reply:
How should that happen? AFAICS, without those there is no code in the cmake build system making the `tests` target depend on the python unit test added.

Reply:
What should we do with this? I'd suggest to add the dependencies back as otherwise the build system might not work correctly.